### PR TITLE
Merge #3823, #3824, #3826, #3827, #3828 to prod

### PR DESCRIPTION
**Pull request by yuvipanda**

### CODE REVIEW

File: hub/templates/nfs-pvc.yaml [hunk 1]
```diff
@@ -29,4 +29,37 @@ spec:
   resources:
     requests:
       storage: 1Mi
+---
```
Comment (ryanlovett) [inline hub/templates/nfs-pvc.yaml:32]: If `useV3` is true, won't there be two pv/pvc? (should the v4 block be in an `else` structure?

**Review by ryanlovett — COMMENTED** [review]
See comment -- should the v4 block be part of an `else`?

File: hub/templates/nfs-pvc.yaml (hunk 1 above)
Comment (ryanlovett) [reply hub/templates/nfs-pvc.yaml:32]: Also, when I try to start a pod on staging I get `pod didn't trigger scale-up: 6 persistentvolumeclaim "home-nfs-v3-v3" not found`.

Comment (yuvipanda) [reply hub/templates/nfs-pvc.yaml:32]: @ryanlovett yeah there will be two. We can't modify PVCs or PVs after creation, so to use multiple options we need multiple ones. And we can't delete PVCs when they are still in use, so if we delete the old ones we'll have to kick out *all* users from the cluster

**Pull request merged**